web-app/src/main/webapp/index.html

```diff
@@ -3,7 +3,9 @@
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="css/skeleton.css">
+    <link rel="stylesheet" href="css/normalize.css">
```

Comment:
nit: I would put in reverse order. Usually more 'fundamental' CSS files go first.